Update am-installing.adoc (#133)
mulesoft/docs-monitoring: modules/ROOT/pages/am-installing.adoc

== What to do Next

> - If you find connection failure or retry entries, ensure that port 443 in your outbound firewall are open.

Reviewer: ```suggestion
- If you find connection failure or retry entries, ensure that port 443 in your outbound firewall is open.
```

> - If you find connection failure or retry entries, ensure that port 443 in your outbound firewall is open. Older runtimes might still use port 5044 and that needs to be open as well, if that's the case.

Reviewer: ```suggestion
- If you find connection failure or retry entries, ensure that port 443 in your outbound firewall is open. If an older runtime uses port 5044, the port must remain open.
```

Reviewer: Are connection failure/retry entries log entries? Should specify throughout the topic.

Author: Not really. If the runtime is using the old port for some reason, we should mention checking both ports (443/5044) to be open

Author: @sarathecoubian Any updates here?

Reviewer 2: @juanpablotimpanaro Click "commit suggestion".

Reviewer: If that change is okay with you, @juanpablotimpanaro , please commit it. This change has been requested by some users over Slack, so we should make sure it gets merged soon.
